.github/workflows/Manager-CI.yaml

```diff
@@ -106,23 +106,23 @@ jobs:
           DEV_WEBHOOK_CLIENT_ID: ${{ secrets.DEV_WEBHOOK_CLIENT_ID }}
           DEV_WEBHOOK_CLIENT_SECRET: ${{ secrets.DEV_WEBHOOK_CLIENT_SECRET }}
         run: |
-          sed -i -E "s|(.*EVENT_BRIDGE_SSO_CLIENT_ID=).*|\1$DEV_EVENT_BRIDGE_SSO_CLIENT_ID|" kustomization.yaml
-          sed -i -E "s|(.*EVENT_BRIDGE_SSO_SECRET=).*|\1$DEV_EVENT_BRIDGE_SSO_SECRET|" kustomization.yaml
-          sed -i -E "s|(.*WEBHOOK_TECHNICAL_ACCOUNT_ID=).*|\1$DEV_WEBHOOK_TECHNICAL_ACCOUNT_ID|" kustomization.yaml
-          sed -i -E "s|(.*WEBHOOK_CLIENT_ID=).*|\1$DEV_WEBHOOK_CLIENT_ID|" kustomization.yaml
-          sed -i -E "s|(.*WEBHOOK_CLIENT_SECRET=).*|\1$DEV_WEBHOOK_CLIENT_SECRET|" kustomization.yaml
+          sed -i -E "s|(.*EVENT_BRIDGE_SSO_CLIENT_ID: ).*|\1'$DEV_EVENT_BRIDGE_SSO_CLIENT_ID'|" event-bridge-shard-operator-secrets.yaml
+          sed -i -E "s|(.*EVENT_BRIDGE_SSO_SECRET: ).*|\1'$DEV_EVENT_BRIDGE_SSO_SECRET'|" event-bridge-shard-operator-secrets.yaml
+          sed -i -E "s|(.*WEBHOOK_TECHNICAL_ACCOUNT_ID: ).*|\1'$DEV_WEBHOOK_TECHNICAL_ACCOUNT_ID'|" event-bridge-shard-operator-secrets.yaml
+          sed -i -E "s|(.*WEBHOOK_CLIENT_ID: ).*|\1'$DEV_WEBHOOK_CLIENT_ID'|" event-bridge-shard-operator-secrets.yaml
+          sed -i -E "s|(.*WEBHOOK_CLIENT_SECRET: ).*|\1'$DEV_WEBHOOK_CLIENT_SECRET'|" event-bridge-shard-operator-secrets.yaml
```

Comment:
I think this is not going to work as you are replacing the Secret data with plaintext: the strings should be b64 encoded

Reply:
I already tested it with Github action. It works just fine.

Reply:
I see you are using `stringData`, fine.